src/Composer/Action/BaseAction.php

```diff
@@ -241,7 +240,7 @@ protected function getPool()
     }
 
     /**
-     * Determine the minimum requried stability.
+     * Determine the minimum reburied stability.
```

Comment:
I … can't … tell … you … how … hard … I … am … laughing … right … now … !

"Welcome to Bolt, we bury our stability in a secret pirate chest on a deserted island …"